src/test/java/lambda/part1/exercise/Exercise3.java

```diff
@@ -0,0 +1,67 @@
+package lambda.part1.exercise;
+
+import com.google.common.collect.FluentIterable;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+public class Exercise3 {
+
+    @Test
+    public void sortPersonsByAgeUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        // TODO использовать Arrays.sort + expression-lambda
+        Arrays.sort(persons, (p1, p2) -> Integer.compare(p1.getAge(), p2.getAge()));
+
+        assertArrayEquals(new Person[]{
+            new Person("Иван", "Мельников", 20),
+            new Person("Николай", "Зимов", 30),
+            new Person("Алексей", "Доренко", 40),
+            new Person("Артем", "Зимов", 45)
+        }, persons);
+    }
+
+    @Test
+    public void sortPersonsByLastNameThenFirstNameUsingArraysSortExpressionLambda() {
+        Person[] persons = getPersons();
+
+        Arrays.sort(persons, (p1, p2) -> p1.getLastName().compareTo(p2.getLastName()) == 0
```

Comment:
You could use `p1.getLastName().equals(p2.getLastName())` instead, it makes easier to read the code. 
Also in this case, you won't get `NullPointerException` if p2's last name is `null`.

Reply:
Thanks. Fixed.